common/spec/dependabot/clients/azure_spec.rb

```diff
@@ -181,6 +181,107 @@
     end
   end
 
+  describe "#create_pull_request" do
+    subject do
```

Comment:
It'd be nice if we gave this subject a name, so we can reference it in the tests:

```suggestion
    subject(:create_pull_request) do
```